Finmer.Game/Gameplay/Character.cs

```diff
@@ -296,6 +296,10 @@ public bool CanSwallow(Character prey)
             return prey.Size <= Size;
         }
 
+        public bool CanGrapple(Character victim) 
```

Comment:
Would you mind adding xmldocs to this function? All public members should have at least a `<summary>` section. Check the above functions (like IsDead, CanSwallow) for examples.

Reply:
Ah, okay, I did that. Hopefully the documentation makes as much sense to you as it does to me.